### ISSUE TITLE
weather map refresh requirement?

### ISSUE THREAD

**Issue by jdcoats** [howardjones/network-weathermap#212]
With cacti 1.2.2 and weathermap dev  If we go to weathermap page and try to use the hover over for any weather map link it does not work. If we hit f5 or refresh and the the hover over link works. I(s there something I missed to make this work without having to f5 every map to get the hover over to work?

**Comment by pautiina** [comment]
I confirm this problem

**Comment by netniV** [comment]
Is this when in full page Weathermaps only? Or when you have the Cacti console/top header still present?

**Comment by howardjones** [comment]
I don't actually remember putting in any special support for hover maps at all yet, in the new version :-)

**Comment by pautiina** [comment]
> Is this when in full page Weathermaps only? Or when you have the Cacti console/top header still present?

With Cacti console/top header

**Comment by howardjones** [comment]
I would guess that overlib.js is not included in the page. The browser javascript console will probably confirm that.

**Comment by pautiina** [comment]
> I would guess that overlib.js is not included in the page. The browser javascript console will probably confirm that.

I checked the browser console, it's empty

**Comment by jdcoats** [comment]
@howardjones yes I remember reading in some of your notes at one time that the hover was not implemented yet, that's why I never worried with asking or reporting. I just happen to stumble on the fact that If I hit f5 they worked again :)
